web/src/components/temp/FormFieldTest.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import * as Yup from 'yup';
+
+import { Input } from '../InputsWithValidation';
+import { updateApd as updateApdAction } from '../../actions/apd';
+
+const FormFieldTest = ({ overview, updateApd }) => (
+  <div className="app">
+    <h3>Form Fields w/ validation</h3>
+    <Input
+      name="email"
+      label="Email"
+      type="email"
+      validation={Yup.string()
```

Comment:
Like it.  We've got `joi` somewhere on the API side but not consistently.  I'm 100% okay with switching.